### PR TITLE
feat: Scrape Radiant Spincrystals

### PR DESCRIPTION
**Pull request by nzeng8**

### CODE REVIEW

File: .github/workflows/update-spincrystals.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+
+      - name: Install missing packages
```
Comment (maalni) [inline .github/workflows/update-spincrystals.yml:26]: Are these needed?

File: .github/workflows/update-spincrystals.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+name: Spincrystals Update
+
+on:
+  schedule:
+    # Runs at 1:00 AM every Thursday every 2 weeks
+    - cron: '0 1 * * 4/2'
```
Comment (maalni) [inline .github/workflows/update-spincrystals.yml:6]: You can run this every day ^^

File: scripts/workflow/update_spincrystal.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            region: cells[3].textContent?.trim() || '',
+        };
+        data.push(spincrystal);
+    }
+    return data;
+};
+
+const main = async () => {
+    const data = await parseData(url);
+    console.log(data);
```
Comment (maalni) [inline scripts/workflow/update_spincrystal.ts:55]: No need to log data here

**Review by maalni — CHANGES_REQUESTED** [review]

File: scripts/workflow/update_spincrystal.ts (hunk 3 above)
Comment (nzeng8) [reply scripts/workflow/update_spincrystal.ts:55]: Oh sorry, forgot to remove

File: .github/workflows/update-spincrystals.yml (hunk 2 above)
Comment (nzeng8) [reply .github/workflows/update-spincrystals.yml:6]: Does it need to be ran every day though? These aren't updated very often (every 2 patches or so)

Comment (maalni) [reply .github/workflows/update-spincrystals.yml:6]: Yeah but if the job misses the changes it has to wait 2 weeks for data to be updated

File: .github/workflows/update-spincrystals.yml (hunk 1 above)
Comment (nzeng8) [reply .github/workflows/update-spincrystals.yml:26]: I'm not really sure, borrowed this from another workflow as I'm not entirely sure how github workflows work. I guess the missing packages could be omitted though.

Comment (maalni) [reply .github/workflows/update-spincrystals.yml]: You would need those packages if you would fetch via bash. Node/Bun already includes dependencies and git should already be installed on the runner

**Review by maalni — APPROVED** [review]

**Pull request merged**